packages/fluxible-addons-react/connectToStores.js

```diff
@@ -59,6 +59,7 @@ function createComponent(Component, stores, getStateFromStores, customContextTyp
     });
 
     hoistNonReactStatics(StoreConnector, Component);
+    StoreConnector.WrappedComponent = Component;
```

Comment:
So I went with capital `WrappedComponent` to align with how [react-intl exposes it](https://github.com/yahoo/react-intl/blob/master/src/inject.js#L32) as well as how [react-router exposes it](https://github.com/reactjs/react-redux/blob/master/docs/api.md#static-properties). However, I see now that the fluxible-router package in this repo is using `wrappedComponent` as you mentioned.

It is a good call out. Does the fluxible team want to align on lowercased within the fluxible project or shift to uppercase in [RouteHandler](https://github.com/yahoo/fluxible/blob/44f39fe1e917ba427b115ec78268c0a18acd8eb7/packages/fluxible-router/lib/handleRoute.js#L55) and [HistoryHandler](https://github.com/yahoo/fluxible/blob/51330c95fab507d47f445ad7d5bf85928fed096f/packages/fluxible-router/lib/handleHistory.js#L289)?